hazelcast/src/main/java/com/hazelcast/internal/monitor/MemberState.java

```diff
@@ -58,34 +51,10 @@
      */
     String getName();
 
-    Map<String, Long> getRuntimeProps();
-
-    LocalMapStats getLocalMapStats(String mapName);
-
-    LocalMultiMapStats getLocalMultiMapStats(String mapName);
-
-    LocalQueueStats getLocalQueueStats(String queueName);
-
-    LocalTopicStats getLocalTopicStats(String topicName);
-
-    LocalTopicStats getReliableLocalTopicStats(String reliableTopicName);
-
-    LocalPNCounterStats getLocalPNCounterStats(String pnCounterName);
-
-    LocalReplicatedMapStats getLocalReplicatedMapStats(String replicatedMapName);
-
-    LocalExecutorStats getLocalExecutorStats(String executorName);
-
-    LocalCacheStats getLocalCacheStats(String cacheName);
-
     LocalWanStats getLocalWanStats(String schemeName);
```

Comment:
It seems this method is not called.

Reply:
Yup, and it was like this before the PR if I'm not mistaken. This whole interface feels artificial.

Removed the method in 3bae9fb45929a28d57d909a33aa7192c11018b94